applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -3,5 +3,182 @@
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``nrf_desktop_main``             | ``module_state_event`` |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``ble_state``                    | ``ble_peer_event``     |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``usb_state``                    | ``usb_state_event``    |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``hid_state``                    | ``hid_report_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``power_manager``                | ``power_down_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+|                                  | ``wake_up_event``      |                                  |                      |                                    |
++----------------------------------+------------------------+----------------------------------+----------------------+------------------------------------+
+
+Configuration
+*************
+
+This module uses Zephyr's :ref:`zephyr:power_management` subsystem.
+It depends on :option:`CONFIG_HAS_SYS_POWER_STATE_DEEP_SLEEP_1` being enabled and selects :option:`CONFIG_DEVICE_POWER_MANAGEMENT` and :option:`CONFIG_SYS_POWER_DEEP_SLEEP_STATES`.
+
+You can enable the power manager module by selecting the ``CONFIG_DESKTOP_POWER_MANAGER_ENABLE`` option in the configuration.
+
+Timeout configuration options
+    With the ``CONFIG_DESKTOP_POWER_MANAGER_TIMEOUT`` configuration option you can set time after which the application will go to low power mode.
+    By default, the timeout is set to 120 seconds.
+
+    The ``CONFIG_DESKTOP_POWER_MANAGER_ERROR_TIMEOUT`` sets the time after which the device is turned off upon an internal error.
+
+Optional idle boolean
+    The ``CONFIG_DESKTOP_POWER_MANAGER_STAY_ON`` lets the system stay idle also when there are no active connections.
```

Comment:
I have changed `idle` to `on` to avoid confusion. I don't want to use names that were yet not introduced. The effect of this option is clearly explained later on.